protocol-configuration/pauseState.ts

```diff
@@ -0,0 +1,30 @@
+export const pauseStateConfig: PauseStateConfig = {
+  ethPSM: {
+    paused: true,
+    redeemPaused: true
+  },
+  lusdPSM: {
+    paused: true,
+    redeemPaused: true
+  },
+  raiPriceBoundPSM: {
+    paused: true,
+    redeemPaused: true,
+    mintPaused: true
+  },
+  daiFixedPricePSM: {
+    paused: false,
+    redeemPaused: false,
+    mintPaused: false
+  }
+};
+
+export type PauseState = {
```

Comment:
Should change this to PSMPauseState, and this should be an interface instead of a type generally.

Reply:
I think we can keep this name generic, there's no reason we wouldn't use this to check paused state of other contracts (in addition to PSMs)

Reply:
Also think this should stay generic. It does the job that we need right now in allowing us to quickly verify pause state with a local config